lib/ziti-tunnel/include/ziti/ziti_tunnel.h

```diff
@@ -94,9 +94,9 @@ typedef host_ctx_t * (*ziti_sdk_host_cb)(void *ziti_ctx, uv_loop_t *loop, const
 
 /** data needed to intercept packets and dial the associated ziti service */
 typedef struct intercept_ctx_s  intercept_ctx_t;
-typedef bool (*intercept_match_addr_fn)(ip_addr_t *addr, void *app_intercept_ctx);
+typedef const ziti_address * (*intercept_match_addr_fn)(ip_addr_t *addr, void *app_intercept_ctx);
```

Comment:
are these changes breaking changes and thus should get a version bump accordingly?

Reply:
Technically it is a breaking change, but as far as I know this API is only used internally (in ziti-tunnel.lib). I'll increment the version to be sure/clear.